core/frontend/src/Viewport.ts

```diff
@@ -2209,7 +2216,9 @@ export abstract class Viewport implements IDisposable {
       this.setupFromView();
 
     if (this._selectionSetDirty) {
+      this._frameStatsCollector.beginTime("hiliteTime");

```

Comment:
Not useful - it just assigns to a variable.

Reply:
Removed.